### ISSUE TITLE
Batter levels are wrong on g533

### ISSUE THREAD

**Issue by Patola** [ashkitten/g933-utils#11]
First of all, thank you very much for these utilities... For us Linux-only users it is a big gift!!

I have a logitech g533 headphone.

Battery levels for the phone when it is charging seems to be measured wrongly. I tried to charge mine and it was at 27%. Then a few seconds after I plugged it into an USB port it showed charging [ascending] at 9.5%. After about 30 minutes, it got to full. But then I unplugged it and it went straight back into... 9% [discharging]? Plugged it in again, 100% [full] immediately. Then changed to 3.5% [charging]. Then to 100% [full], then 3.5% [charging] again. It's going back and forth between 3.5% and 100% now. RUST_BACKTRACE set to 1 doesn't show anything.

Additionally, I know I should be using a separate ticket for that, but I was never able to use g933-utils set equalizer 6 7 5 4 2 0 0 0 1 2 -- it always times out after 6 seconds. I can use "get equalizer" though.

**Comment by ashkitten** [comment]
hi! thanks for submitting an issue, i'll take a look at it. i don't have a g533, so i'll try to troubleshoot blindly with you. i've seen a behavior with my g933 where it will incorrectly report voltage and thus battery level immediately after plugging or unplugging the headset, does that seem to be the same issue you're experiencing?

**Comment by Patola** [comment]
Sorry for taking that long to respond. Well, it seems so, the fluctuating levels are indeed right after plugging or unplugging. Note: I will look more often at my github responses, won't take much to answer again

**Comment by ashkitten** [comment]
yeah so right now i'm just naively translating immediate voltage reports to the data points on the discharge graph. without keeping some sort of persistent state, there's not a whole lot i can do to make it more stable i think. i'll gladly review a pull request, but i haven't had the energy to work on this project much lately (and i still have some other stuff left to do whenever i do find a good time to do things)

**Comment by Patola** [comment]
I understand. Although I am a professional programmer, I confess, my life is a mess and I don't have time for everything I want. Unfortunately, I don't have enough free time to help you on that. But many thanks for making the code public. Let's hope someone else can do it at some time.

**Comment by ashkitten** [comment]
heh, relatable. good luck, i'll leave the issue open
